### PR TITLE
Fix run in docker

### PR DESCRIPTION
**Pull request by ivdunin**
1. Fixed docker-compose.yml
2. Fixed Ruby image
3. Fixed dot.Net image
4. Fixed typo for Flask port

### CODE REVIEW

**Comment by ivdunin** [comment]
@albedoweb @chetvil all quickstart app tested. All works

**Review by albedoweb — APPROVED** [review]

**Comment by ivdunin** [comment]
@chetvil could you please review it

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
 To get started with Citadel API, we recommend following a step-by-step walkthrough in our docs https://docs.citadelid.com/docs/web-quickstart.
+
+or run all-in-one docker-compose
+```shell
+cd quickstart
+make env
+docker-compose up -d
+```
+
+check integration for your programming language:
```
Comment (chetvil) [inline README.md:11]:
```suggestion
Check the quickstart demo for your programming language:
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+1. http://localhost:3000 Ruby
```
Comment (chetvil) [inline README.md:12]: @ivdunin can we make all ports sequential? 5001, 5002....

**Comment by ivdunin** [comment]
@chetvil done

**Review by chetvil — APPROVED** [review]

**Pull request merged**